clients/python/girder_client/__init__.py

```diff
@@ -387,7 +378,7 @@ def sendRestRequest(self, method, path, parameters=None,
             parameters = {}
 
         # Look up the HTTP method we need
-        f = self.METHODS[method]
```

Comment:
We can store the `Girder-Token` header within the session object when we set the token (via `self.session.headers.update`) so we don't have to do the merging logic with `_headers`.